include/uprotocol-cpp/uri/serializer/MicroUriSerializer.h

```diff
@@ -148,7 +148,7 @@ class MicroUriSerializer {
       * @param resource 
       * @return 
       */
-    [[nodiscard]] [[maybe_unused]] auto isMicroForm(const uprotocol::v1::UEntity &entity) -> bool {
+    [[nodiscard]] [[maybe_unused]] static auto isMicroForm(const uprotocol::v1::UEntity &entity) -> bool {
```

Comment:
Do not use "auto" except for obtaining a function return.  Please use the known type of bool.  This happens many times in this file.  Remove "-> bool".  No need for lambda here.

Reply:
Sure @MarkStokes71, i will convey the info to @MelamudMichael.